packages/storage/src/indexed-db/index.ts

```diff
@@ -422,7 +422,14 @@ export class IndexedDb implements IndexedDbInterface {
     const allGasPrices = await this.db.getAll('GAS_PRICES');
     return allGasPrices
       .map(gp => GasPrices.fromJson(gp))
-      .filter(gp => !gp.assetId?.equals(this.stakingTokenAssetId));
+      .filter(
+        gp =>
+          !gp.assetId?.equals(this.stakingTokenAssetId) &&
+          (gp.blockSpacePrice > 0n ||
+            gp.compactBlockSpacePrice > 0n ||
+            gp.verificationPrice > 0n ||
+            gp.executionPrice > 0n),
```

Comment:
question: what does this mean? We should not accept genesis gas prices?

Reply:
I was concerned about a scenario where a governance proposal for `feeParams` could pass that adds an asset with zero gas prices. In that context, wouldn't users be able to transact with zero gas fees for that asset?

not sure what you mean by rejecting genesis gas prices?

Reply:
> I was concerned about a scenario where a governance proposal for feeParams could pass that adds an asset with zero gas prices. In that context, wouldn't users be able to transact with zero gas fees for that asset?

Even if that happens, we probably shouldn't override the community's election

Reply:
sure good call, removed